rest/specs/src/main/filtered-resources/openapi/openapi.yaml

```diff
@@ -58,17 +58,17 @@ paths:
         '501':
           $ref: '#/components/responses/NotImplemented'
   # TODO This endpoint will be replaced
-  /product/{productId}/{metadataType}:
+  /dataset/{datasetId}/{metadataType}:
     put:
-      summary: Add metadata to a Product.
+      summary: Add a metadata to a dataset.
```

Comment:
Add a metadata ***attachment*** to a dataset.

Reply:
What value does adding the word `attachment` add?

Reply:
makes it more readable. only a suggestion

Reply:
I'm afraid of adding the word attachment here and it becoming a first-level term like `file`, `metadata`, and `dataset`